package.json

```diff
@@ -21,6 +21,8 @@
     "got": "^10.7.0",
     "graphql": "^14.6.0",
     "hypixelconstants": "^3.2.1",
+    "mean-average": "^1.0.0",
```

Comment:
I don't think adding new dependencies to replace existing functions is the way to go since there are really no benefits and only more clutter to package.json

Reply:
@builder-247 Fixed!